src/content/release-notes.mdx

```diff
@@ -5,6 +5,27 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.35.0
+
+1 August 2025
+
+This version is compatible with Kubernetes versions 1.31 to 1.33
+Okteto Chart release 1.35 is designed to work with [Okteto CLI 3.10.x](https://github.com/okteto/okteto/releases/tag/3.10.0)
+
+### New Features {#new-features-1.35}
+- CIDR-based traffic filtering is now supported for BYOC (Bring Your Own Cluster) environments. Use this to improve security by restricting access to Okteto services to specific IP ranges.
```

Comment:
Is this something we want to mention in the release notes taking into account we didn't do changes in the chart more than the point about supporting `loadBalancerSourceRanges` in BuildKit?

Customers in BYOC model cannot do the changes on their own. Maybe this is something to add into some BYOC specific instead of here?

If we add it in the docs, I think it is important to highlight the restrictions (I don't know the details, but I do know there are some)